src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+@import "./utils/variables.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__description {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $color-secondary;
+    align-self: flex-start;
+  }
+
+  &__rewiews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-value {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-main-accent;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__value {
+    font-size: 12px;
+    font-weight: 400;
+    color: $color-secondary;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    color: $color-main-accent;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: flex;
+    height: 40px;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+    font-weight: 700;
+    line-height: 16px;
+    font-size: 14px;
+    text-transform: uppercase;
+    color: white;
+    background-color: $color-accent-blue;
+    border-radius: 5px;
+    border: 1px solid transparent;
+  }
+
```

Comment:
I suppose it would be better to do like this:

Reply:
```suggestion
border: 1px solid transparent;
 &:hover {
    color: $color-accent-blue;
    background-color: #fff;
    border-color: $color-accent-blue;
      }
    }
```

Reply:
Done

Reply:
Good job!